### ISSUE TITLE
kiall -f minor issue

### ISSUE THREAD

**Issue by impulse200** [HewlettPackard/LinuxKI#43]
Hello. 
I was wondering why on my machine `kiall -f' doesn't work with messages:
```
[sergey@win7 work]$ kiall -f
find: unknown predicate `--header'
kiall complete
```
It is the line [504](https://github.com/HewlettPackard/LinuxKI/blob/49daa9172a76d3ad6f16ec49cb2aaaed43b8b6eb/scripts/kiall#L504) of the script with find command gives such output.
Looking carefully at the files in current directory I found this gem:
```
sergey@win7 work]$ ll
total 712
-rw-rw-r--. 1 sergey sergey 130547 июн 15 21:39 boot-1.log
[... skip ...]
-rw-rw-r--. 1 sergey sergey    131 июн 12 15:36 --header
[... skip ...]
```
The fix is simple: just replace '*' to '.' in find parameters. I'm not sure if such trivial thing is worth creating pull requests and all this suff...

**Comment by MarkCRay** [comment]
Thanks.   This will be fixed in the 7.1 version of LinuxKI, which I hope to release in a week or 2.

**Comment by MarkCRay** [comment]
Fixed in LinuxKI version 7.1, released on 07/09/21
